usvm-core/src/main/kotlin/org/usvm/utils/ResultUtils.kt

```diff
@@ -0,0 +1,9 @@
+package org.usvm.utils
+
+import org.usvm.solver.USatResult
+import org.usvm.solver.USolverResult
+
+fun <T> USolverResult<T>.assertSat(): USatResult<T> {
```

Comment:
As for me, it is better to rename it as `ensureSat`. The reason, is that `assert` is closely related to SMT solver assert operation.

Reply:
Fair enough